frappe/__init__.py

```diff
@@ -52,8 +52,30 @@ def _(msg, lang=None):
 	if lang == "en":
 		return msg
 
-	return get_full_dict(local.lang).get(msg) or msg
+	# To parse html data
+	msg = parse_msg(msg)
```

Comment:
Source has to be copy-pasted from HTML in the translation table.

Here you are solving only one use case, what about other use-cases. And you cannot assume whitespace like `\n'

Remove this code.